lib/generateNormals.js

```diff
@@ -14,10 +17,7 @@ module.exports = generateNormals;
  * @param gltf
  */
 function generateNormals(gltf) {
-    var primitives = getPrimitives(gltf);
-    var primitivesLength = primitives.length;
-    for (var i = 0; i < primitivesLength; i++) {
-        var primitive = primitives[i];
+    jp.apply(gltf, '$.meshes[*].primitives[*]', function (primitive) {
```

Comment:
Have you looked at microbenchmarks?  How fast is this compared to doing it by hand and how much GC pressure does it create?  Given how general this library is and how specific our needs are, I suspect we can do something much better ourselves with very little work.  Even if we use this library, as our needs grow, we'll end up writing custom helper functions.